### ISSUE TITLE
GRASS fails with Unable to reach inflow error message

### ISSUE THREAD

**Issue by landam** [storm-fsv-cvut/smoderp2d#176]
```sh
./bin/grass/r.smoderp2d/r.smoderp2d.py elevation=dem10m@PERMANENT soil=soils@PERMANENT soil_type_fieldname=SID vegetation=landuse@PERMANENT vegetation_type_fieldname=LandUse rainfall_file=tests/data/rainfall.txt maxdt=30 end_time=40 points=points@PERMANENT table_soil_vegetation=soil_veg_tab_mean@PERMANENT table_soil_vegetation_fieldname=soilveg streams=stream@PERMANENT channel_properties_table=stream_shape@PERMANENT streams_channel_type_fieldname=channel_id output=tests/data/output
```

fails with

```
Start of computing...
ERROR: Unable to reach inflow. Feature id 5 not found in 1,3,2
```

**Comment by landam** [comment]
time_step.py:

```
               surface_state = surface.arr.get_item([i, j]).state

                if surface_state > Globals.streams_flow_inc:
                    # stream flow in the cell

                    surface.arr.get_item([i, j]).h_total_new = 0.0

                    h_sub = subsurface.runoff_stream_cell(i, j)

                    inflowToReach = h_sub * pixel_area + surBIL * pixel_area

                    surface.reach_inflows(
                        surface_state - Globals.streams_flow_inc,
                        inflowToReach)
```

**Comment by pesekon2** [comment]
@landam: Isn't this fixed by #177?

**Comment by pesekon2** [comment]
> @landam: Isn't this fixed by https://github.com/storm-fsv-cvut/smoderp2d/pull/177?

Fixed with #177
